Core/Security/X509Chain.py

```diff
@@ -30,7 +36,13 @@ def __init__( self, certList = False, keyObj = False ):
     self.__hash = False
     if certList:
       self.__loadedChain = True
-      self.__certList = certList
+      self.__certList = []
+      for cert in certList:
+        if isinstance( cert, M2Crypto.X509.X509 ):
```

Comment:
if NOT isiinstance

Reply:
![image](https://user-images.githubusercontent.com/1897515/31339418-192fce22-ad03-11e7-895f-fef8a2888c0c.png)